theory/tests/test_periodic.c

```diff
@@ -579,6 +579,7 @@ int main(int argc, char **argv)
     options.fast_divide_and_NR_steps=0;
     options.copy_particles=1;
     options.float_type=sizeof(double);
+    options.boxsize = boxsize;
```

Comment:
Where is the `boxsize` on the RHS defined?

Reply:
tests_common.h.  I think it's only valid for some of the datasets, but that's why I added the `allboxsize` array.